projects/storefrontlib/shared/components/item-counter/item-counter.component.spec.ts

```diff
@@ -212,7 +218,7 @@ describe('ItemCounterComponent', () => {
         By.css('button')
       );
       expect(
-        (<HTMLButtonElement>button[0].nativeElement).disabled
+        (<HTMLButtonElement>button[0].nativeElement).ariaDisabled
```

Comment:
Looks like there are other tests that verified the button disabling behaviour. Let's make sure those are updated as well.

Reply:
I've updated some other test cases too.